test/io/test_tim_grabber.cpp

```diff
@@ -0,0 +1,97 @@
+#include <pcl/test/gtest.h>
+#include <pcl/point_types.h>
+#include <pcl/io/pcd_io.h>
+#include <pcl/io/tim_grabber.h>
+
+#include <string>
+#include <random>
+
+using PointT = pcl::PointXYZ;
+using CloudT = pcl::PointCloud<PointT>;
+
+class TestableTimGrabber : public pcl::TimGrabber {
+  using TimGrabber::TimGrabber;
+  public:
+		using TimGrabber::point_cloud_xyz_ptr_;
+		using TimGrabber::processTimPacket;
+		using TimGrabber::updateLookupTables;
+		using TimGrabber::toPointClouds;
+};
+
+std::vector<std::string> packets_;
+std::vector<CloudT> correct_clouds_;
+TestableTimGrabber grabber_;
```

Comment:
Is there any benefit to these variables being global vs them being members of TimGrabberTest class?